src/main/java/frc/robot/Constants.java

```diff
@@ -57,4 +57,13 @@ public class Controller {
         public static final int LEFT_JOYSTICK_BUTTON = 9;
         public static final int RIGHT_JOYSTICK_BUTTON = 10;
     }
+
+    public class Undertaker {
+        public static final int INTAKE_MOTOR_ONE = 13;
+        public static final int INTAKE_MOTOR_TWO = 14;
+
+        public static final double INTAKE_SPEED = 6.0;
```

Comment:
lets rename these `INTAKE_VOLTAGE` and `EJECT_VOLTAGE` to better describe what they are